vars/postgresql_10.yml

```diff
@@ -0,0 +1,6 @@
+---
+# PostgreSQL vars for v10
+
+postgresql_autovacuum_vacuum_cost_delay: 20ms
+
+...
```

Comment:
@gclough TIL that `...` may be provided at the end of a YAML file.

I wonder though, if it wouldn't be better to remove them (which I believe is done elsewhere), because I'm pretty sure this will surprise a lot of people!

(now maybe you are using a linter of some sort etc!)

Reply:
Yes, sorry... it's my habit to use correctly formed YAML, but it doesn't match the style of the role so I'll remove it.  Thanks for the input @thbar